src/sagemaker/session.py

```diff
@@ -671,6 +684,10 @@ def endpoint_from_job(self, job_name, initial_instance_count, instance_type,
             wait (bool): Whether to wait for the endpoint deployment to complete before returning (default: True).
             model_environment_vars (dict[str, str]): Environment variables to set on the model container
                 (default: None).
+            model_vpc_config (dict[str, list[str]]): Overrides VpcConfig set on the model.
```

Comment:
Tom mentioned he wants us to use a marker like USE_MODEL_VPC. Are we still doing that?

Reply:
Tom clarified that he was asking for a global constant like this which would be used as the default value. It makes it more clear for a user to reuse training's `vpc_config`, or else override to some different value, or even `None`. I will add this.